pkg/apis/serving/v1alpha1/revision_validation.go

```diff
@@ -135,15 +134,26 @@ func validateContainer(container corev1.Container) *apis.FieldError {
 	return errs
 }
 
-func validateContainerPorts(ports []corev1.ContainerPort) bool {
-	bIsAllowed := false
-	// users can set container port which names "user-port" to define application's port.
+func validateContainerPorts(ports []corev1.ContainerPort) *apis.FieldError {
+	// user can set container port which names "user-port" to define application's port.
 	// Queue-proxy will use it to send requests to application
-	if len(ports) == 1 && ports[0].Name == "user-port"{
-		bIsAllowed = true
-		return bIsAllowed
+	// if user didn't set any port, it will set default port user-port=8080.
+	if len(ports) > 1 {
+		return &apis.FieldError{
+			Message: "container ports set more than one",
```

Comment:
Change to "More than one container ports is set"?

Reply:
ok